From: Markus Pargmann <mpa@pengutronix.de>
Date: Mon, 17 Nov 2014 10:56:46 +0100
Subject: [PATCH] configure: Remove introspection checks

This patch is not for mainline. We just don't care about introspection.

Signed-off-by: Markus Pargmann <mpa@pengutronix.de>
---
 configure.ac | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5acbcd193e61..3d5640126fa9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1585,12 +1585,6 @@ if test -n "$export_dynamic"; then
 fi
 
 ##################################################
-# GObject introspection
-##################################################
-
-GOBJECT_INTROSPECTION_CHECK([0.9.3])
-
-##################################################
 # Checks for gtk-doc and docbook-tools
 ##################################################
 
